test_cases/bin/recovery_test_wrapper.sh

```
@@ -0,0 +1,89 @@
+
+set -e
+
+if [ $# -lt 1 ]; then
+    echo "ERROR: recovery_test_wrapper.sh requires TASK_FILE parameter" >&2
+    echo "Usage: recovery_test_wrapper.sh TASK_FILE [OPTIONS...]" >&2
+    exit 1
+fi
+
+TASK_FILE="$1"
+shift
+OPTIONS="$@"
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)